openapi/keap/types.bal

```diff
@@ -0,0 +1,1741 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
```suggestion
// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Resolved all the suggestions. Thank you.